packages/backend/src/open_payments/payment/outgoing/worker.ts

```diff
@@ -63,16 +64,24 @@ async function handlePaymentLifecycle(
   deps: ServiceDependencies,
   payment: OutgoingPayment
 ): Promise<void> {
-  if (payment.state !== OutgoingPaymentState.Sending) {
-    deps.logger.warn('unexpected payment in lifecycle')
-    return
-  }
+  const tracer = trace.getTracer('outgoing_payment_worker')
```

Comment:
actually, let us also include the code where we actually fetch the outgoing payment from the DB (it's happening in `getPendingPayment`)